src/docs/home.md

```diff
@@ -1,3 +1,9 @@
 # Introduction
```

Comment:
I'm not sure we need this -- we could just use "The Harmonized CRDC Data Model (CRDC-H)" as the top-level header on this page?

Reply:
Oh, that's a good point. I can remove, if you don't manage to merge before I get there.... 
oh.. hehe. too late. 😄 ...

Reply:
Haha, sorry!